doc/dev.md

```diff
@@ -31,19 +31,69 @@ For quicker tests, while developing, try `./scripts/run test`.
 
 ## Publishing
 
+### Configure gpg for signing the release
```

Comment:
This is great - thank you for taking the time to research & document this so well - looks really useful!